spec.bs

```diff
@@ -3479,6 +3533,41 @@ A <dfn>reporting result</dfn> is a [=struct=] with the following [=struct/items=
      {{InterestGroupReportingScriptRunnerGlobalScope/registerAdBeacon(map)}}.
 </dl>
 
+TODO: Remove the following once https://github.com/WICG/turtledove/pull/771 is merged.

```

Comment:
this block till the next todo is copied from PR 771. Going to remove the block once that PR is merged.